test/common_utils.py

```diff
@@ -589,76 +624,90 @@ class BoundingBoxLoader(TensorLoader):
     spatial_size: Tuple[int, int]
 
 
-def randint_with_tensor_bounds(arg1, arg2=None, **kwargs):
-    low, high = torch.broadcast_tensors(
-        *[torch.as_tensor(arg) for arg in ((0, arg1) if arg2 is None else (arg1, arg2))]
+def make_bounding_box(
+    size=None,
+    *,
+    format=datapoints.BoundingBoxFormat.XYXY,
+    spatial_size=None,
+    batch_dims=(),
+    dtype=None,
+    device="cpu",
+):
+    def sample_position(values, max_value):
+        # We cannot use torch.randint directly here, because it only allows integer scalars as values for low and high.
+        # However, if we have batch_dims, we need tensors as limits.
+        return torch.stack([torch.randint(max_value - v, ()) for v in values.flatten().tolist()]).reshape(values.shape)
+
+    if isinstance(format, str):
+        format = datapoints.BoundingBoxFormat[format]
+
+    if spatial_size is None:
+        if size is None:
+            spatial_size = DEFAULT_SIZE
```

Comment:
I'm a bit confused by this logic here and mostly have 2 Qs:

- Do we ever call `make_bounding_box()` with both size and spatial_size to None?
- If we do, shouldn't the logic be: 1) set `size` to `DEFAULT_SIZE` (for consistency with the rest of the helpers) and 2) generate `spatial_size` accordingly?

Reply:
> My logic here was that if we want to generate an arbitrary sample, we can just do

OK why not - maybe just add that as a comment then